lib/ui/sign_in.dart

```diff
@@ -139,6 +138,7 @@ class SignInWidgetState extends State<SignInWidget> {
                                 //Store the credentials into the the secure storage only if validated
                                 if (loggedIn) {
                                   Keychain().addCredentials(email, password);
+                                  await CacheService.updateMail();
```

Comment:
this is also already happening in main.dart, not sure where would be better but right now it'd happen twice

Reply:
My understanding is that it only happens in `main.dart` if there are login credentials saved. If that is the case, the cache is populated and the user is directed to the main menu. Otherwise, the cache is not populated and the user is directed to this sign in page. In order for the cache to populate here, we have to call `updateMail` in this path as well.

I may have that wrong tho! So let me know if I'm tracking this wrong.

Reply:
ah missed that, thanks!